src/main/java/uk/gov/hmcts/darts/common/datamanagement/api/DataManagementFacade.java

```diff
@@ -10,12 +17,11 @@
  */
 public interface DataManagementFacade {
```

Comment:
I'm not clear on whether we are expected to support the link to `case_document` here as it exists in our Table "darts.external_object_directory".

It might be out of scope / something missed?
What do you think @hemantasharma1129